src/CssManager.php

```diff
@@ -950,7 +950,7 @@ function setCSSboxshadow($v)
 					}
 				}
 			}
-			if (!isset($new['col'])) {
+			if (!empty($new['col'])) {
```

Comment:
Oh, sorry. Its automatically error. I'll fix it in the next commit (other `isset` too).